_docs/collection-details.md

```diff
@@ -3,8 +3,8 @@ title: Collection details
 permalink: /docs/collection-details/
 ---
 
-* Table of contents
-{:toc}
+- Table of contents
+  {:toc}
```

Comment:
@hillarioh This change broke the Table of contents and should be reverted.
![Screenshot 2022-05-26 at 12 28 17](https://user-images.githubusercontent.com/4462641/170460229-6ee75120-7434-4332-9ba8-2dffb093ebf8.png)